docs/ocean/leap_authorization.rst

```diff
@@ -10,40 +10,92 @@ and higher enable Leap\ |tm| authentication using the secure
+    run one of the following commands from your terminal. If your account has
+    more than one project, the :ref:`tab_authorize_multiple_leap_projects` tab
+    below shows how to select one.
+
+    .. tab-set::
+
+        .. tab-item:: Single Leap Project
+            :name: tab_authorize_single_leap_project
+            :selected:
+
+            .. include:: ../shared/authorization.rst
+                :start-after: start_local_system
+                :end-before: end_local_system
+
+            .. doctest::
+                :skipif: True
+
+                dwave setup --auth
+
+            .. include:: ../shared/authorization.rst
+                :start-after: start_cloud_system
+                :end-before: end_cloud_system
+
+            .. doctest::
+                :skipif: True
+
+                dwave setup --oob
+
+        .. tab-item:: Multiple Leap Projects
+            :name: tab_authorize_multiple_leap_projects
+
+            You can log in to your |cloud|_ account, set the preferred
+            project to be the active project (by selecting
+            *your_user_name* > **Projects** > *project*) and then use the
+            procedure described in the :ref:`tab_authorize_single_leap_project`
+            tab.
 
-    -   In an environment such as an **IDE installed on your system**, where you
-        can access `localhost <https://en.wikipedia.org/wiki/Localhost>`_
-        addresses from your browser, initiate the OAuth *redirect* flow::
+            Alternatively, you can use the ``--project`` option to select a
+            project as shown here (the example code below selects a project
+            named ``PROJ3``):\ [#]_
 
-            dwave auth login
+            .. include:: ../shared/authorization.rst
+                :start-after: start_local_system
+                :end-before: end_local_system
 
-    -   In an environment such as a **cloud IDE**, where access to
-        `localhost <https://en.wikipedia.org/wiki/Localhost>`_ addresses from
-        your browser might be blocked, you can initiate the alternative OAuth
-        *out-of-band* flow::
+            .. doctest::
+                :skipif: True
 
-            dwave auth login --oob
+                dwave setup --auth --project PROJ3
+
+            .. include:: ../shared/authorization.rst
+                :start-after: start_cloud_system
+                :end-before: end_cloud_system
+
+            .. doctest::
+                :skipif: True
+
+                    dwave setup --oob --project PROJ3
+
+            You can always switch your environment to use the API token of
+            another project later. For example, if you ran the procedure of the
+            :ref:`tab_authorize_single_leap_project` tab with the ``PROJ2``
+            project set to active in Leap, you can at any time switch to the
+            ``PROJ3`` project with the
+            ``dwave config create --auto --project PROJ3`` command.
```

Comment:
I would keep `--auto-token` instead `--auto`. It's clearer what part of config create is "auto".

Reply:
Updated